--- ltris-1.2.7/Makefile.am.orig	2023-10-04 01:55:10.271766701 +0100
+++ ltris-1.2.7/Makefile.am	2023-10-04 01:55:34.559743827 +0100
@@ -8,12 +8,12 @@ CROSSCOMPILE ltris.appdata.xml
 install-data-local:
 	$(mkinstalldirs) $(DESTDIR)$(datadir)/icons/hicolor/apps
 	$(mkinstalldirs) $(DESTDIR)$(datadir)/applications
-	$(INSTALL_DATA) ltris48.png $(DESTDIR)$(datadir)/icons/hicolor/apps/ltris.png
+	$(INSTALL_DATA) ltris48.png $(DESTDIR)$(datadir)/icons/ltris.png
 	$(INSTALL_DATA) ltris.desktop $(DESTDIR)$(datadir)/applications
 	
 
 uninstall-local:
-	rm $(DESTDIR)$(datadir)/icons/hicolor/apps/ltris.png
+	rm $(DESTDIR)$(datadir)/icons/ltris.png
 	rm $(DESTDIR)$(datadir)/applications/ltris.desktop
 
 ACLOCAL_AMFLAGS = -I m4
